tests/functional/test_article.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+
+from pages.article import ArticlePage
+
+
+# page title
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_is_title_expected(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.is_title_expected()
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_is_title_in_title(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.is_title_in_title()
```

Comment:
As above, as assert for substr in str would give a more informative trace on failure.